runtimebp/signal.go

```diff
@@ -0,0 +1,41 @@
+package runtimebp
+
+import (
+	"context"
+	"os"
+	"os/signal"
+)
+
+// ShutdownHandler is the callback type used in HandleSignals.
+type ShutdownHandler func(signal os.Signal)
+
+var defaultSignals = []os.Signal{
+	os.Interrupt,
+	os.Kill,
```

Comment:
I wonder if those two should be in the same default one. Usually interrupt waits for whatever is still being processed to finish and kill terminates them all immediately.

Reply:
Good point. Updated the default list.